celestia/NakoTurk.txt

```diff
@@ -0,0 +1 @@
+1d824f2cf4df9c9da98c120b78be3f4319a1d7ff@46.4.23.42:26656
```

Comment:
this should be added to the peers.txt in the same directory and not its own file. I would look at what other PRs are doing, but if its easier adding this text could be a separate PR

Reply:
@evan-forbes this was requested by the team thats why he is making the PR like that

Reply:
yes @mindstyle85 you are correct, this should be fine
the CI should pass with you individual peer.txt file

Reply:
other PRs are already passing with this config.

Reply:
Solved, Thank you

Reply:
cool, this makes sense, approving!